pkg/telemetryquerylanguage/functions/tqlcommon/README.md

```diff
@@ -27,6 +28,22 @@ Examples:
 
 - `IsMatch("string", ".*ring")`
 
+## Int
+
+`Int(value)`
+
+The `Int` factory function converts a float, bool and string data to the int type.
```

Comment:
Can you explicitly list out the types converted, and explicitly state that the returned type is int64